crates/gen-host/src/lib.rs

```diff
@@ -349,7 +396,9 @@ impl Host {
                 get_cx: impl Fn(&T) -> &U + Send + Sync + 'static,
             ) -> Result<(), ::tauri_bindgen_host::ipc_router_wip::Error>
             where
+                T: Send + Sync + 'static,
                 U: #trait_ident + Send + Sync + 'static,
+                R: ::tauri_bindgen_host::tauri::Runtime
```

Comment:
there is no R, if I remove it the example works fine. (But I'm not yet advanced enough to fully understand the whole process here)
Also all tests are failing because they expect
```
pub fn add_to_router<T, U, R>(
        router: &mut ::tauri_bindgen_host::ipc_router_wip::Router<T, R>,
```

Reply:
Yep you're right! Leftover from when it was passing the AppHandle down but that's not needed anymore